tests/specs/tram-spec.js

```diff
@@ -71,6 +72,18 @@ const tests = (Tram) => describe('Tram', () => {
     })
   })
 
+  describe('addListener', () => {
+    it('should call single listener', () => {
+      const spy = jasmine.createSpy('listener')
+      const app = new Tram()
+      app.addActions({counter: counterActions})
+      app.engine.addListener(spy)
```

Comment:
we should call `app.addListener(spy)` since that is the function we are exposing.